R/weight.R

```diff
@@ -153,8 +153,9 @@ generateWeightEntry <- function(expr) {
         target_list[[i]] <- c(i, targets[i])
     }
 
-    list(
+    out <- list(
         variable = self(var),
         targets = target_list
     )
+    out
```

Comment:
This should be `return(out)`

Reply:
Can you say more about the reasoning? Do we always require an explicit return statement?